src/interface/widgets/TreeVisualizerWidget.cpp

```diff
@@ -98,7 +98,9 @@ void TreeVisualizerWidget::populateRow(bt::Node::Ptr node, QTreeWidgetItem* row,
     QString status = QString::fromStdString(node->status_print(node->getStatus()));//statusToString(node->getStatus()));
     if (row->text(1)!=status) {
         row->setText(1, status);
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
I personally do not like ignoring these warnings. These functions are deprecated and therefore need replacement instead of being ignored

Reply:
Exactly. they need to be but im not familiar with Qt5 and I couldn't quickly find a function that satisfy the requirements for this call, I temporarily suppressed it, I thought i added a comment saying they had to be replaced but I guess not :)

I'm completely against any warnings, if it were up to me I'd add -Wall -Wextra -Werror -pedantic to compiler flags.